waiter/integration/waiter/health_check_test.clj

```diff
@@ -252,3 +252,25 @@
                 (is (= (str "Hello " (retrieve-username)) (-> backend-response :body str))))))
           (finally
             (delete-token-and-assert waiter-url token)))))))
+
+(deftest ^:parallel ^:integration-fast test-ping-deployment-error
```

Comment:
Can we introduce assertions in the tests here: https://github.com/twosigma/waiter/blob/master/waiter/integration/waiter/deployment_errors_test.clj

That way we do not have to worry about the overhread of launching a new service and detecting it is failing.